_posts/2024-06-03-so-you-want-better-opensearch-documentation.markdown

```diff
@@ -0,0 +1,60 @@
+
+![lightning talk](../assets/media/blog-images/2024-06-03-improvedocs/2024-06-03-improvedocs.jpg)(https://www.youtube.com/watch?v=eRLmIzevC-4)
+
+We started with the slightly tongue-in-cheek statement that the way you search Reddit is to go to Google and search “_How do I (insert any task here) +Reddit_” to get the information you want. The pattern that lots of folks use to find information about OpenSearch (unsurprisingly, as OpenSearch started as a fork of Elasticsearch) is to go to Google and search “_How do I (insert any task here) +Elasticsearch”_. That statement was met with a few nodding heads and light chuckling.  
+
+We then introduced [_The Documentation System_](https://docs.divio.com/documentation-system/), which is referred to as *The Grand Unified Theory of Documentation*. You may know it as [_Diátaxis_](https://diataxis.fr/)*,* (I am unclear about the relationship between the two sites!) Here’s a [video](https://www.youtube.com/watch?v=t4vKPhjcMZg) explaining the concept, which divides documentation into four quadrants: Tutorials, How-To Guides, Explanation, and Reference. 
+
+![Divio quadrants](../assets/media/blog-images/2024-06-03-improvedocs/2024-06-03-quadrant.jpg)
+
+OpenSearch actually does a pretty good job of the _Reference_ portion of the puzzle, especially considering that there are new releases every six weeks. I started contributing documentation late last year and recently joined the [_opensearch-project/documentation-website_](https://github.com/opensearch-project/documentation-website) repository as a Maintainer. I see a constant flow of both new docs and the improvement of existing docs. Considering the ratio of writers to developers and where the project was not too long ago, I think we’re doing a really good job of keeping up with both. 
+
+The other three quadrants, however, is where we could do a better job. There is _Tutorial_  and _Explanation_ content mixed in with the _Reference_ content, but you can spend a lot of time wading through the minutiae of how things work when you are really looking for understanding of the core principles. We also have a huge amount of great content in the existing [blog](https://opensearch.org/blog/) that could flesh out these other areas of documentation, not to mention if we start linking out to third-party blogs. This is an area I plan to invest more energy into.
+
+Charlie then asked the audience what their thoughts were and we captured a lot of great insights. I then followed up with Heather Halter whose remarks are in *italics*. 
```

Comment:
"questions and comments" rather than "insights" completes the logical flow from the intro into the Q&A.